### PR TITLE
Include ROSA customer-initiated incident documentation

### PR DESCRIPTION
**Pull request by wgordon17**
Version(s):
`enterprise-4.14+`

Issue:

Link to docs preview:
- [Incident Management](https://71403--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix#rosa-policy-incident-management_rosa-policy-responsibility-matrix)
- [Cluster capacity](https://71403--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix#rosa-policy-cluster-capacity_rosa-policy-responsibility-matrix)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/rosa-policy-incident.adoc [hunk 1]
```diff
@@ -109,6 +109,13 @@ When managing a new incident, Red Hat uses the following general workflow:
 . The incident is documented and a root cause analysis (RCA) is performed within 5 business days of the incident.
 . An RCA draft document will be shared with the customer within 7 business days of the incident.
 
+Red Hat is also available to assist with customer incidents when raised through a support case.
```
Comment (jneczypor) [inline modules/rosa-policy-incident.adoc:112]:
```suggestion
Red Hat also assists with customer incidents raised through support cases. 
```

File: modules/rosa-policy-incident.adoc [hunk 2]
```diff
@@ -109,6 +109,13 @@ When managing a new incident, Red Hat uses the following general workflow:
 . The incident is documented and a root cause analysis (RCA) is performed within 5 business days of the incident.
 . An RCA draft document will be shared with the customer within 7 business days of the incident.
 
+Red Hat is also available to assist with customer incidents when raised through a support case. 
+Red Hat is able to assist with activities including but not limited to:
```
Comment (jneczypor) [inline modules/rosa-policy-incident.adoc:113]:
```suggestion
Red Hat assists with activities including but not limited to:
```

File: modules/rosa-policy-incident.adoc [hunk 3]
```diff
@@ -109,6 +109,13 @@ When managing a new incident, Red Hat uses the following general workflow:
...
+
+- Forensic gathering, including isolating virtual compute
```
Comment (jneczypor) [inline modules/rosa-policy-incident.adoc:115]:
```suggestion
* Forensic gathering, including isolating virtual compute
```

File: modules/rosa-policy-incident.adoc [hunk 4]
```diff
@@ -109,6 +109,13 @@ When managing a new incident, Red Hat uses the following general workflow:
...
+- Guiding compute image collection
```
Comment (jneczypor) [inline modules/rosa-policy-incident.adoc:116]:
```suggestion
* Guiding compute image collection
```

File: modules/rosa-policy-incident.adoc [hunk 5]
```diff
@@ -109,6 +109,13 @@ When managing a new incident, Red Hat uses the following general workflow:
...
+- Providing collected audit logs
```
Comment (jneczypor) [inline modules/rosa-policy-incident.adoc:117]:
```suggestion
* Providing collected audit logs
```

**Review by jneczypor — COMMENTED** [review]
Just a few minor edits for style. Looks good.

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.16` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.15. And any PR going into `main` must also target the latest version branch (enterprise-4.16).

If the update in your PR does NOT apply to version 4.16 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by dramseur** [comment]
@EricPonvelle @jneczypor  - Please review / approve this PR. Thanks!

**Comment by EricPonvelle** [comment]
Hi @dramseur, I still see the suggestions from @jneczypor. Can we resolve the comments that won't be done, and then I can merge it?

**Comment by dramseur** [comment]
@wgordon17 - -Please see the suggestions from [@jneczypor](https://github.com/jneczypor). 
Please let @EricPonvelle  now if that comment can be resolved since it won't be done..and then the PR can be merged.

File: modules/rosa-policy-incident.adoc (hunk 2 above)
Comment (wgordon17) [reply modules/rosa-policy-incident.adoc:113]: What about 

```suggestion
Red Hat can assist with activities including but not limited to:
```

I'm just trying to make it more obvious that we're not providing this assistance automatically or proactively, but that we _can_ provide this assistance when requested

Comment (jneczypor) [reply modules/rosa-policy-incident.adoc:113]: I think the "can" works nicely.

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**